Gemfile.lock

```diff
@@ -2,15 +2,25 @@ PATH
   remote: .
   specs:
     clean-architecture (2.3.1)
+      activemodel (>= 5)
+      activesupport (>= 5)
```

Comment:
Urgh... I don't love this creeping into this codebase

Reply:
Yeah agreed, I think perhaps a `CleanArchitectureOnRails` gem might find itself being needed at some point to encapsulate this sort of stuff but for now I think `activemodel` and `activesupport` are widely used enough that its not too big a deal